### PR TITLE
Add version: KDE.rkward version 0.7.5

### PR DESCRIPTION
**Pull request by superusercode**
###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/107600)

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:
image

**Comment by stephengillie** [comment]
Manual Validation encountered:
image

**Comment by stephengillie** [comment]
Using the `Custom` switch `/AllUsers` bypasses this switch, but doesn't result in a silent install:
image

File: manifests/k/KDE/rkward/0.7.5/KDE.rkward.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+PackageIdentifier: KDE.rkward
+PackageVersion: 0.7.5
+InstallerType: nullsoft
+FileExtensions:
+- r
+ReleaseDate: 2022-10-20
+Installers:
+- Architecture: x64
+  InstallerUrl: https://download.kde.org/stable/rkward/0.7.5/rkward-0.7.5-R-4.2.0-KF5-5.98.0.exe
```
Comment (stephengillie) [inline manifests/k/KDE/rkward/0.7.5/KDE.rkward.installer.yaml:12]:
```suggestion
  InstallerUrl: https://download.kde.org/stable/rkward/0.7.5/rkward-0.7.5-R-4.2.0-KF5-5.98.0.exe
  Scope: machine
```

File: manifests/k/KDE/rkward/0.7.5/KDE.rkward.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  InstallerSha256: 0A93736D158EE4BF5E8BD7921A7BB4FC207D459631A6CA27C4912EAD9E03FD73
```
Comment (stephengillie) [inline manifests/k/KDE/rkward/0.7.5/KDE.rkward.installer.yaml:13]:
```suggestion
  Scope: machine
  InstallerUrl: https://download.kde.org/stable/rkward/0.7.5/rkward-0.7.5-R-4.2.0-KF5-5.98.0.exe
  InstallerSha256: 0A93736D158EE4BF5E8BD7921A7BB4FC207D459631A6CA27C4912EAD9E03FD73
  InstallerSwitches:
    Custom: /AllUsers
```

**Comment by ItzLevvie** [comment]
Close with reason: Stale pull request;

**Pull request closed without merging**